airbyte-integrations/connectors/source-looker/unit_tests/test_url.py

```diff
@@ -0,0 +1,46 @@
+import re
+import pytest
+
+def valid_url(test_str):

```

Comment:
also, this still does not prevents us from duplication. What will happen with this test if we decide to change the spec?

Reply:
you can check airbyte-integrations/connectors/source-freshdesk/integration_tests/test_client.py
for examples